corehq/motech/repeaters/models.py

```diff
@@ -1342,6 +1342,12 @@ class RepeatRecordManager(models.Manager):
     def count_pending_records_for_domain(self, domain):
         return self.filter(domain=domain, next_check__isnull=False).count()
 
+    def count_overdue(self, threshold=timedelta(minutes=10)):
+        return self.filter(
+            next_check__isnull=False,
+            next_check__lt=datetime.utcnow() - threshold
+        ).count()
```

Comment:
Looks like the argument is never passed, so the default is always used. I copied this from the dbaccessor. It does seem like it could be useful in a shell context so I'm going to leave it.